src/controllers/midi/midicontroller.h

```diff
@@ -112,6 +114,27 @@ class MidiController : public Controller {
     // So it can access sendShortMsg()
     friend class MidiOutputHandler;
     friend class MidiControllerTest;
+    friend class MidiControllerJSProxy;
+};
+
+class MidiControllerJSProxy: public ControllerJSProxy {
+  public:
+    MidiControllerJSProxy(MidiController* m_pController)
+    : ControllerJSProxy(m_pController),
+      m_pController(m_pController) {
+    }
+
+    Q_INVOKABLE void sendShortMsg(unsigned char status,
+            unsigned char byte1, unsigned char byte2) {
+        m_pController->sendShortMsg(status, byte1, byte2);
+    }
+
+    Q_INVOKABLE void sendSysexMsg(QList<int> data, unsigned int length = 0) {
+        m_pController->sendSysexMsg(data, length);
+    }
+
+  private:
+    MidiController* m_pController;
```

Comment:
```suggestion
    MidiController* m_pMidiController;
```
to avoid confusion with `m_pController` in the parent class